src/components/SudokuGrid/Cell.js

```diff
@@ -1,10 +1,22 @@
-export default function Cell({ value, onCellClick, id, isEditable }) {
+export default function Cell({
+  value,
+  onCellClick,
+  id,
+  isEditable,
+  isSelected,
+}) {
   function handleClick() {
     onCellClick(id);
   }
 
   return (
-    <button onClick={handleClick} disabled={isEditable ? false : true}>
+    <button
+      className={`Cell ${isEditable ? "" : "Cell--locked"} ${
+        isSelected ? "Cell--active" : ""
+      } ${`CellRow${id[0]}`} ${`CellColumn${id[1]}`}`}
```

Comment:
Maybe you could write a function that returns the classNames. This is hard to read and to understand. Simple abstract this for better readability.

Reply:
I will do that, thanks for the advice:)